docs/usage/key-concepts/how-renovate-works.md

```diff
@@ -0,0 +1,161 @@
+---
+title: How Renovate works
+description: Learn how Renovate works
+---
+
+# Introduction
+
+Renovate usually performs these steps:
+
+- Clones the repo
+- Scans package files to extract dependencies
+- Looks up registries to check for available updates
+- Applies any grouping rules defined
+- Pushes branches and raises Pull Requests
+
+Because Renovate needs to support a lot of dependency naming and versioning conventions, it has modules for each known convention.
+You can contribute your own modules, if you want.
+
+## Modules
+
+Renovate's modules are:
+
+- [manager](../modules/manager/index.md)
+- [datasource](../modules/datasource/index.md)
+- [versioning](../modules/versioning.md).
```

Comment:
Sort alphabetically, add platform too. We also need to make sure such links work when published to docs

Cc @HonkingGoose